README.md

```diff
@@ -6,80 +6,7 @@ The common library of TERASOLUNA Server Framework for Java (5.x) is a library of
-  </tr>
-  <tr>
-    <td>1</td>
-    <td>terasoluna-gfw-common</td>
-    <td>Functionality that can be used in general and not just related to the web</td>
-    <td>Yes</td>
-    <td>
-    <pre>
-* Comprehensive Common Exception Handling Mechanism
-  * General exception class (designed as per the needs of this mechanism)
-  * Exception Logger
-  * Exception Codes
-  * Interceptor to output exception log
-* Improvised System Time handling mechanism (using java.util.Date, java.sql.Date, java.sql.Time, java.sql.Timestamp)
-* Codelist functionality
-* Improvised Message handling mechasim
-* Query excaping utilities for SQL, JPQL
-* Sequencer classes
-     </pre>
-    </td>
-  </tr>
-  <tr>
-    <td>2</td><td>terasoluna-gfw-jodatime</td><td>Maven dependency definition and functionality using Joda Time</td><td>Yes</td><td><pre>* Improvised System Time handling mechanism (using org.joda.time.DateTime)</pre></td>
-  </tr>
-  <tr>
-    <td>3</td>
-    <td>terasoluna-gfw-web</td><td>Functionalities that will be useful while developing a web application</td>
-    <td>Yes</td>
-    <td>
-    <pre>
-* Transaction Token Mechanism (Mechanism to prevent double submit)
-* Common Exception Handler
-* Interceptor to load Codelist
-* Download View
-* Group of Servlet filters to output log of information in MDC
-  * Servlet filter parent class
-  * Servlet filter to output tracking Id
-  * Servlet filter to clear MDC
-* Group of EL functions
-  * Counter-measure for Cross-Site-Scripting
-  * URL encoding functionality
-  * Creating query parameters from JavaBean
-* JSP tag to display pagination
-* JSP tag to display output messages after request processes
-    </pre>
-    </td>
-  </tr>
-  <tr>
-    <td>4</td><td>terasoluna-gfw-mybatis3</td><td>Maven dependency definition for MyBatis3</td><td>No</td><td><pre>* Dependency definition for MyBatis3</pre></td>
-  </tr>
-  <tr>
-    <td>5</td><td>terasoluna-gfw-jpa</td><td>Maven dependency definition for JPA</td><td>No</td><td><pre>* Dependency definition for JPA</pre></td>
-  </tr>
-  <tr>
-    <td>6</td><td>terasoluna-gfw-security-core</td><td>Maven dependency definition for using spring-security (other than web)</td><td>No</td><td><pre>* Dependency definition for Spring Security (other than web)</pre></td>
-  </tr>
-  <tr>
-    <td>7</td>
-    <td>terasoluna-gfw-security-web</td>
-    <td>Maven dependency definition for using spring-security (web related) and components that extend spring-security</td>
-    <td>yes</td>
-    <td>
-    <pre>
-* Servlet filter to output the authenticated username in log
-* Redirect handler to counter-measure open redirect vulnerablibility
-    </pre>
-    </td>
-  </tr>
-</table>
+The following project are included in TERASOLUNA Server Framework for Java (5.x) Common Library, refer to [TERASOLUNA Server Framework for Java (5.x) Common Library](https://github.com/terasolunaorg/guideline/blob/master/source_en/Overview/FrameworkStack.rst#building-blocks-of-common-library).
```

Comment:
- [ ] Modify as following

All modules included in TERASOLUNA Server Framework for Java (5.x) Common Librariey are listed up in the [development guideline](https://github.com/terasolunaorg/guideline/blob/master/source_en/Overview/FrameworkStack.rst#building-blocks-of-common-libra).